Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -72,13 +71,7 @@ public static class Forms
 		static Color _ColorButtonNormal = Color.Default;
 		public static Color ColorButtonNormalOverride { get; set; }
 
-		internal static BuildVersionCodes SdkInt {
-			get {
-				if (!s_sdkInt.HasValue)
-					s_sdkInt = Build.VERSION.SdkInt;
-				return (BuildVersionCodes)s_sdkInt;
-			}
-		}
+		internal static BuildVersionCodes SdkInt => Anticipator.SdkInt;
```

Comment:
The UIThread and the thread pool threads will race to place this call. Usually the thread pool thread will win and the UIThread will be saved the effort of crossing into the android OS get get the sdk version.